base_wamas/security/ir_model_access.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
```

Comment:
@tuantrantg any particular reasons to make access rights noupdate?

Reply:
hi a @dzungtran89 actually not :D I just forget to remove it
--> It'll removed in the upcoming commit